src/renderers/shared/fiber/ReactFiberUpdateQueue.js

```diff
@@ -76,7 +76,11 @@ exports.callCallbacks = function(queue : UpdateQueue, context : any) {
     if (callback && !node.callbackWasCalled) {
       node.callbackWasCalled = true;
       if (typeof context !== 'undefined') {
-        callback.call(context);
+        try {
+          callback.call(context);
+        } catch (error) {
+          // do something
```

Comment:
Could you please add actual logic to handle the error? You can look for how we do this in `ReactFiberCommitWork.js` when calling lifecycle methods.